CHANGELOG.md

```diff
@@ -12,6 +12,13 @@ in the naming of release branches.
 
 ### Added
 
+- Add ToExpr (from tree-diff) instances for all types inside NewEpochState
+  - Add module Cardano.Ledger.TreeDiff in cardano-binary. Includes all needed operations, classes, and orphan instances	
```

Comment:
```suggestion
- Add ToExpr (from tree-diff) instances for all types inside NewEpochState
  - Add module Cardano.Ledger.TreeDiff in cardano-data. Includes all needed operations, classes, and orphan instances	
```

Reply:
we didn't move this, so this suggestion is moot.

Reply:
That's right, however there still a small fix is needed: `cardano-binary` -> `cardano-ledger-binary`